hardened-binaries/test.sh

```diff
@@ -8,7 +8,7 @@ echo ".NET Core base directory: $root"
 
 # TODO handle more architectures can just x86-64
 
-file_list=$(find "$root/" -type f -exec file {} \; | grep 'ELF 64-bit LSB' | cut -d: -f 1 | sort -u)
+file_list=$(find "$root/" -type f -exec file {} \; | grep -E 'ELF 64-bit [LM]SB' | cut -d: -f 1 | sort -u)
```

Comment:
`file` reports `LSB` on x86_64 and reports `MSB` on s390x (and other big-endian systems).